### PR TITLE
Change state when clicking added-services list item

### PR DESCRIPTION
**Pull request by hatched**
When clicking the added services list items it should change the state so that the inspector for that service is shown.

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

**Comment by hatched** [comment]
#### To QA

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2887/
Test PASSed.

File: jujugui/static/gui/src/app/components/added-services-list/added-services-list.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ YUI.add('added-services-list', function() {
 
     generateItemList: function(services) {
       var items = [];
-      services.forEach(function(service) {
+      services.forEach((service) => {
```
Comment (makyo) [inline jujugui/static/gui/src/app/components/added-services-list/added-services-list.js]: I, for one, welcome our new arrowfunc overlords.

**Comment by makyo** [comment]
:+1: will QA after lunch

File: jujugui/static/gui/src/app/components/added-services-list/added-services-list.js (hunk 1 above)
Comment (hatched) [inline jujugui/static/gui/src/app/components/added-services-list/added-services-list.js]: :-)

Comment (mitechie) [inline jujugui/static/gui/src/app/components/added-services-list/added-services-list.js]: no f'ing way...did he actually arrow func? Is that even allowed? That works in IE10?

**Comment by makyo** [comment]
QA okay

File: jujugui/static/gui/src/app/components/added-services-list/added-services-list.js (hunk 1 above)
Comment (hatched) [inline jujugui/static/gui/src/app/components/added-services-list/added-services-list.js]: Not at chance :) 

Babel converts it to a bound function in the output - yay for a build process!

Comment (hatched) [inline jujugui/static/gui/src/app/components/added-services-list/added-services-list.js]: @mitechie It converts it to...

```JavaScript
      var _this = this;

      services.forEach(function (service) {
          ...
          changeState: _this.props.changeState,
          ...
      });
```

**Comment by anthonydillon** [comment]
Code and QA looks good :+1:

**Comment by hatched** [comment]
Thanks guys! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1018

**Comment by hatched** [comment]
Looks to be an npm error

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**